tests/integration/test_s3.py

```diff
@@ -176,12 +176,12 @@ def test_can_delete_urlencoded_object(self):
         bucket_contents = self.client.list_objects(
             Bucket=self.bucket_name)['Contents']
         self.assertEqual(len(bucket_contents), 1)
-        self.assertEqual(bucket_contents[0]['Key'], 'a+b/foo')
+        self.assertEqual(bucket_contents[0]['Key'], u'a+b/foo')
```

Comment:
Why do you need to add the unicode prefix to these? I feel like these should still pass with the unicode prefixed removed.